wpilibcExamples/src/main/cpp/examples/ElevatorSimulation/cpp/Robot.cpp

```diff
@@ -37,6 +38,14 @@ class Robot : public frc::TimedRobot {
   static constexpr int kJoystickPort = 0;
 
   static constexpr double kElevatorKp = 5.0;
+  static constexpr double kElevatorKi = 0.0;
+  static constexpr double kElevatorKd = 0.0;
+
+  static constexpr double kElevatorkS = 0.0;
+  static constexpr double kElevatorkG = 0.0;
+  static constexpr double kElevatorkV = 0.762;
+  static constexpr double kElevatorkA = 0.762;
```

Comment:
These need to be in the proper units using the unit library- see [the constructor](https://github.com/wpilibsuite/allwpilib/blob/main/wpimath/src/main/native/include/frc/controller/ElevatorFeedforward.h#L31-L42) for the proper types.

Reply:
Sorry, could you clarify what you mean by them having to be in proper units?

Reply:
They need to use the C++ unit library, instead of being doubles. This could be done here or in the feedforward constructor.